python_modules/libraries/dagster-databricks/dagster_databricks/configs.py

```diff
@@ -502,10 +502,16 @@ def _define_submit_run_fields():
         "This token should have at most 64 characters.",
         is_required=False,
     )
+    install_default_libraries = Field(
+        Bool,
+        description="To install default libraries or not.",
```

Comment:
"By default, Dagster installs a version of dagster, dagster-databricks, and dagster-pyspark matching the locally-installed versions of those libraries. If you would like to disable this behavior, this value can be set to False."